caesars-palace/blocks/product-details/product-details.js

```diff
@@ -85,6 +107,168 @@ export default async function decorate(block) {
         main.prepend(heroSection);
       }
 
+      /** Create Product quick facts section */
+      const productQuickFactsSection = document.createElement('div');
+      productQuickFactsSection.classList.add('product-quickfacts');
+      // Add the logo and description column block
+      if (logo && longDescription && title) {
+        const logoPicture = createOptimizedPicture(`${logo}`, title, false);
+        const quickFactsBlock = buildBlock('columns', [[logoPicture, longDescription]]);
+        quickFactsBlock.classList.add('product-quickfacts', 'stretch-end');
+        productQuickFactsSection.append(quickFactsBlock);
+      }
+
+      // Add hours
+      const { hours } = json;
+      if (hours && hours.data && (hours.data.length > 0)) {
+        const hoursArray = [];
+        hours.data.forEach((row) => {
+          const day = row.Day;
+          const openTime = getDateFromExcel(row['Open Time']);
+          const openTimeOutput = openTime.toLocaleTimeString('en-US', {
```

Comment:
It's so nice we can do that now that the data comes from Excel. Will simplify the product quickfacts element eventually.